Xamarin.Forms.Core/Shell/Shell.cs

```diff
@@ -130,6 +130,40 @@ static void OnFlyoutBehaviorChanged(BindableObject bindable, object oldValue, ob
 			BindableProperty.CreateAttached("UnselectedColor", typeof(Color), typeof(Shell), Color.Default,
 				propertyChanged: OnColorValueChanged);
 
+		public static readonly BindableProperty BadgeTextColorProperty =
```

Comment:
why do you define them as attached on Shell ?

Reply:
Moved to BaseShellItem.